src/app/components/listing/details/ics-family/ics-family.jsx

```diff
@@ -140,137 +170,152 @@ function ChplIcsFamily(props) {
+                      {' '}
+                      {' '}
+                      <Link
+                        href={compare}
+                        external={false}
+                      >
+                        compare all.
+                      </Link>
+                    </Typography>
+                    { isShowingListingDetails
+                  && (
+                    <div className={classes.detailContainer}>
+                      <Divider />
+                      <Typography variant="h5"><strong>CHPL Product Number:</strong></Typography>
+                      <ChplLink
+                        href={`#/listing/${listing?.id}?panel=additional`}
+                        text={listing?.chplProductNumber}
+                        external={false}
+                      />
                       <Typography>
+                        <strong>Developer:</strong>
                         <ChplLink
-                          href={`#/listing/${listing?.id}?panel=additional`}
-                          text="View full details"
+                          href={`#/organizations/developers/${listing?.developer.id}`}
+                          text={listing?.developer.name}
                           external={false}
                         />
                       </Typography>
-                    )}
-                  <Button
-                    onClick={() => setIsShowingListingDetails(false)}
-                  >
-                    Hide Details
-                  </Button>
-                </>
-              )}
-          </>
+                      <Typography>
+                        <strong>Product:</strong>
+                      </Typography>
+                      <Typography>{ listing?.product.name }</Typography>
+                      <Typography>
+                        <strong>Version:</strong>
+                      </Typography>
+                      <Typography>
+                        { listing?.version.name }
+                      </Typography>
+                      <Typography><strong>Certification Status:</strong></Typography>
+                      <Typography>
+                        { listing?.certificationStatus.name }
+                      </Typography>
+                      <br />
+                      <Button
+                        endIcon={<CloseIcon />}
+                        size="small"
+                        variant="contained"
+                        color="secondary"
+                        onClick={() => setIsShowingListingDetails(false)}
+                      >
+                        Close Details
+                      </Button>
+                    </div>
+                  )}
+                  </div>
+                  <CytoscapeComponent
+                    elements={elements}
+                    style={{ width: '50%', height: '40vh' }}
+                    minZoom={1}
+                    maxZoom={1}
```

Comment:
Why did these change? There was a lot of work that went into getting them right before. Are these changes necessary, and have you compared the change in zoom on a few different trees of ICS?

Reply:
I took out the zoom levels so I user wouldn't get loss within the chart. It very easy to scroll with a mouse and lose your position. We can add them back but I think the functionally isnt helpful

Reply:
So you disabled the zoom? If we want to do that, that's fine, but then we should do it with the actual "disable zoom" configuration. Have you looked at a Listing with a lot of inheritance? I found being able to zoom out to see all of the items, even if you couldn't read the text, was sometimes useful. I also don't really want to remove functionality that's been out there for a while without getting confirmation that we should